Lib/test/test_cmd_line.py

```diff
@@ -46,7 +46,11 @@ def test_site_flag(self):
 
     def test_usage(self):
         rc, out, err = assert_python_ok('-h')
-        self.assertIn(b'usage', out)
+        lines = out.splitlines()
+        self.assertIn(b'usage', lines[0])
+        # The first line contains the program name,
+        # but the rest should be ASCII-only
+        b''.join(lines[1:]).decode('ascii')
```

Comment:
Would decoding each line separately be faster?

Reply:
No, decoding one large string at once is faster.